Code/Editor/EditorViewportWidget.cpp

```diff
@@ -2970,7 +2970,12 @@ void EditorViewportWidget::RestoreViewportAfterGameMode()
 {
     Matrix34 preGameModeViewTM = m_preGameModeViewTM;
 
-    QString text = QString("You are exiting Game Mode. Would you like to restore the camera in the viewport to where it was before you entered Game Mode?<br/><br/><small>This option can always be changed in the General Preferences tab of the Editor Settings, by toggling the \"%1\" option.</small><br/><br/>").arg(EditorPreferencesGeneralRestoreViewportCameraSettingName);
+    QString text =
+        QString(
+            "When leaving \" Game Mode \" the engine will automatically restore your camera position to the default position before you "
+            "had entered Game mode.<br/><br/><small>If you dislike this setting you can always change this anytime in the global "
+            "preferences.</small><br/><br/>")
```

Comment:
Nit: We might want to place a tr() around it already.

Reply:
Fixed!